roles/pulp_api/handlers/main.yml

```diff
@@ -1,9 +1,18 @@
 ---
-# handlers file for pulp_api
+# handlers file for pulp_api service
```

Comment:
This comment is an auto-generated line.

And these are not 2 separate files, so I think you should not touch this comment nor add the other one. The handler names are self-descriptive.